onnxruntime/core/mlas/lib/platform.cpp

```diff
@@ -475,6 +477,8 @@ Return Value:
     this->GemmDoubleKernel = MlasDgemmKernel;
     this->QuantizeLinearS8Kernel = MlasQuantizeLinearS8Kernel;
     this->QuantizeLinearU8Kernel = MlasQuantizeLinearU8Kernel;
+    this->QuantizeLinearS16Kernel = MlasQuantizeLinearS16Kernel;
+    this->QuantizeLinearU16Kernel = MlasQuantizeLinearU16Kernel;
```

Comment:
You need to define MlasQuantizeLinear*16Kernel in [https://github.com/microsoft/onnxruntime/blob/af80542e65334979d3e37c1cb6c88068c5c65f3b/onnxruntime/core/mlas/lib/power/QuantizePower.cpp#L117](https://github.com/microsoft/onnxruntime/blob/af80542e65334979d3e37c1cb6c88068c5c65f3b/onnxruntime/core/mlas/lib/power/QuantizePower.cpp#L117) to support Power arch

Reply:
If you don't want to, you can add opt to use the default C++ implementation